src/NuGet.Core/NuGet.CommandLine.XPlat/Utility/MSBuildAPIUtility.cs

```diff
@@ -323,10 +323,12 @@ internal static bool IsPackageReferenceProject(Project project)
         /// <param name="projectPath"> Path to the project to get versions for its packages </param>
         /// <param name="userInputFrameworks">A list of frameworks</param>
         /// <param name="assetsFile">Assets file for all targets and libraries</param>
-        /// <param name="transitive">Include transitive packages in the result</param>
-        /// <returns></returns>
+        /// <param name="transitive">Include transitive packages/projects in the result</param>
+        /// <param name="includeProjects">Include project references in top-level and transitive package lists</param>
+        /// <returns>FrameworkPackages collection with top-level and transitive package/project
+        /// references for each framework, or null on error</returns>
         internal IEnumerable<FrameworkPackages> GetResolvedVersions(
-            string projectPath, IEnumerable<string> userInputFrameworks, LockFile assetsFile, bool transitive)
+            string projectPath, IEnumerable<string> userInputFrameworks, LockFile assetsFile, bool transitive, bool includeProjects)
```

Comment:
The projects are *only* listed in `dotnet list package`? 

outdated, deprecated and vulnerable don't list them, is that correct?

Reply:
> The projects are _only_ listed in `dotnet list package`?

That's correct, the default report.

> outdated, deprecated and vulnerable don't list them, is that correct?

Yes.